### PR TITLE
Improve drive stats

### PR DESCRIPTION
**Pull request by jheredianet**
This is an improvement suggested here: https://github.com/teslamate-org/teslamate/issues/4119#issuecomment-2265585512

I've taken the opportunity of reorganizing the panels and information, and also optimized the performance of the SQL queries.
As an addition now we can exclude some locations in the top 10 destinations statistics. 

image

### CODE REVIEW

File: grafana/dashboards/drive-stats.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
       "type": "grafana",
       "id": "grafana",
       "name": "Grafana",
-      "version": "11.0.0"
+      "version": "11.1.0"
```
Comment (JakobLichterfeld) [inline grafana/dashboards/drive-stats.json:14]: We are on grafana 11.0.1

File: grafana/dashboards/drive-stats.json [hunk 2]
```diff
@@ -144,7 +118,7 @@
         "textMode": "value",
         "wideLayout": true
       },
-      "pluginVersion": "11.0.0",
+      "pluginVersion": "11.1.0",
```
Comment (JakobLichterfeld) [inline grafana/dashboards/drive-stats.json:121]: We are on grafana 11.0.1

File: grafana/dashboards/drive-stats.json [hunk 3]
```diff
@@ -257,18 +232,19 @@
...
```
Comment (JakobLichterfeld) [inline grafana/dashboards/drive-stats.json:235]: We are on grafana 11.0.1

File: grafana/dashboards/drive-stats.json [hunk 4]
```diff
@@ -345,7 +339,7 @@
...
```
Comment (JakobLichterfeld) [inline grafana/dashboards/drive-stats.json:342]: We are on grafana 11.0.1

File: grafana/dashboards/drive-stats.json [hunk 5]
```diff
@@ -468,7 +463,7 @@
...
```
Comment (JakobLichterfeld) [inline grafana/dashboards/drive-stats.json:466]: We are on grafana 11.0.1

File: grafana/dashboards/drive-stats.json [hunk 6]
```diff
@@ -581,7 +577,7 @@
...
```
Comment (JakobLichterfeld) [inline grafana/dashboards/drive-stats.json:580]: We are on grafana 11.0.1

File: grafana/dashboards/drive-stats.json [hunk 7]
```diff
@@ -669,7 +666,7 @@
...
```
Comment (JakobLichterfeld) [inline grafana/dashboards/drive-stats.json:669]: We are on grafana 11.0.1

File: grafana/dashboards/drive-stats.json [hunk 8]
```diff
@@ -782,7 +780,7 @@
...
```
Comment (JakobLichterfeld) [inline grafana/dashboards/drive-stats.json:783]: We are on grafana 11.0.1

File: grafana/dashboards/drive-stats.json [hunk 9]
```diff
@@ -895,7 +894,7 @@
...
```
Comment (JakobLichterfeld) [inline grafana/dashboards/drive-stats.json:897]: We are on grafana 11.0.1

File: grafana/dashboards/drive-stats.json [hunk 10]
```diff
@@ -989,18 +987,19 @@
         "sizing": "auto",
         "valueMode": "color"
       },
-      "pluginVersion": "11.0.0",
+      "pluginVersion": "11.1.0",
```
Comment (JakobLichterfeld) [inline grafana/dashboards/drive-stats.json:990]: We are on grafana 11.0.1

**Review by JakobLichterfeld — COMMENTED** [review]
Thank you so much! I do like the addition of excluding some location if desired.

**Comment by jheredianet** [comment]
Grafana version is fixed now. Thanks.

**Review by JakobLichterfeld — APPROVED** [review]
lgtm

**Pull request merged**